lib/flop_phoenix.ex

```diff
@@ -388,6 +414,8 @@ defmodule Flop.Phoenix do
     `<tbody>`. Default: `#{inspect(@default_table_opts[:tbody_tr_attrs])}`.
   - `:tbody_td_attrs`: Attributes to added to each `<td>` tag within the
     `<tbody>`. Default: `#{inspect(@default_table_opts[:tbody_td_attrs])}`.
+  - `:event_name`: If set, tells `Flop.Phoenix` to use event based sorting.
```

Comment:
It might be necessary to specify the `phx_target` as well. Maybe we can name the options `live_event` and `live_target`?